crypto/evp/p_lib.c

```diff
@@ -1158,6 +1159,18 @@ int EVP_PKEY_print_params(BIO *out, const EVP_PKEY *pkey,
                       pctx);
 }
 
+static void find_md_nid(const char *mdname, void *data)
```

Comment:
The name of the function is slightly confusing as it does not do the find by itself. Perhaps `mdname2nid` or something similar would be better name?

Reply:
Renamed as suggested.